Packages/PackageSources/meridian-simcache/Data/simcache.base.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "1.0.0",
+    "cache_definitions": [
+        {
+            "meta": {
+                "id": "5b37c963-a842-4277-b932-cc7d56b35f6e",
+                "authors": [
+                    "1a032b37-a56f-43e8-a644-c17d45d6f45c"
+                ],
+                "date_placed": "2024-04-01T02:38:00Z",
+                "difficulty": 3,
+                "simobject_title": "SimCache"
+            },
+            "loc_text": [
+                {
+                    "locale": "en-US",
+                    "text": {
+                        "title": "The Letter C",
+                        "description": "You'll find this letter nestled in the hills on north side of a small canyon named after a red fruit.",
```

Comment:
Sorry, this is probably my typo from when we were originally discussing this -- can we change it to `on the north side` instead?  (I accidentally omitted "the" when initially writing the description.)